pkg/api/v1/routes/handlers.go

```diff
@@ -251,40 +224,44 @@ func (r *Routes) serverConditionCreate(c *gin.Context) {
 	// check if any condition with exclusive set is in incomplete states
 	if errEx := r.exclusiveNonFinalConditionExists(c.Request.Context(), serverID); errEx != nil {
 		if errors.Is(errEx, ErrConditionExclusive) {
-			c.JSON(
-				http.StatusBadRequest,
-				&v1types.ServerResponse{Message: errEx.Error()},
-			)
+			r.logger.WithFields(logrus.Fields{
+				"error": err,
+				// XXX: would be nice to have the id of the exclusive condition here too
```

Comment:
:thumbsup

Reply:
cf. [issue 13](https://github.com/metal-toolbox/conditionorc/issues/13). I will likely take that on as a small follow up to this PR.